cicd/osm/translation/nordic_tags.py

```diff
@@ -24,5 +26,7 @@ def filterTags(attrs):
         "dog": {0: "no", 1: "yes"}[int(attrs.get("dog_friend", 0))],
         "lit": {0: "no", 1: "yes"}[int(attrs.get("lights", 0))],
         f"{piste_prefix}grooming": "classic;skating",
-        "network": "BV Nordic Centre",
+        f"{bvnordic_prefix}network": "Bulkley Valley Nordic Centre",
```

Comment:
I don’t think we want to prefix this tag, I assume nordic pulse is looking for just “network” not “bvnordic:network"

Reply:
Sure, I didn't think this was a Nordic Pulse tag and assumed if was just one of ours. Easy fix!